src/content/getting-started/designers/designers.mdx

```diff
@@ -92,12 +92,14 @@ You can see a full breakdown of the 2x Grid at [IBM.com/design](https://www.ibm.
 
 *Symbols* are Sketch versions of Carbon [components](/components/overview), [add-ons](/contributing/add-ons), and [icons](/guidelines/iconography/library).
 
-<ImageComponent cols="8" caption="Carbon symbol menu navigation">
+<ImageComponent cols="8" caption="IDL symbol menu navigation">
 
-![Carbon symbol menu navigation](/images/symbol-nav.png)
+![Carbon symbol menu navigation](/images/IDL-kit-library.png)
 
 </ImageComponent>
 
+
+
 There are two kinds of symbols — `library symbols` and `document symbols`. Library symbols are available in any Sketch document, while document symbols are specific to the document in which they are found.
 
 From the main menu select `Insert → Symbols → Carbon Design System` to add symbols onto your page. 
```

Comment:
should be "From the `Insert` menu, select `Symbols → Carbon Design System` to add symbols onto your page.

Reply:
@aagonzales is this were we should say something about 'don't disconnect Symbols bc now they are so much more flexible yada yada yada'?

Reply:
@chrisconnors-ibm Thanks for looking at this. In the same vein as this Q for Anna, is my language correct regarding layer styles? 

"If you elect to override a preset layer style, use the existing override styles found in the right-hand panel."

Reply:
People shouldn't be over-riding layer styles or breaking symbols. Once we have the other themes in there we can say something about how you should switch to the correct theme symbols and not over-ride layer styles. We should emphasis not detaching form symbols. 

Somewhere we can do a section on editing the sketch file to update to your own custom theme but out of the box IBMers should not be messing with layer overrides. In fact, we've actually locked a lot of those layers and don't even allow users to edit them. The would have to unlock it on the master to edit.

Reply:
I misunderstood how the layer styles were categorized and corrected the language.